### ISSUE TITLE
Group Bar Graphs Dynamically

### ISSUE THREAD

**Issue by olivinesguerra** [danielgindi/Charts#2158]
I would like to  ask to compute on how to make the bar graphs group dynamically. The json is parse correctl, but its not group correctly. Is there any problem on how I used "[data groupBarsFromX: minimumValue groupSpace: groupSpace barSpace: barSpace];" ? Thanks. I will also attached the .m file and the response from the logs. Thanks

1.Two Bar Graphs - http://prntscr.com/e8398s

[Response.txt](https://github.com/danielgindi/Charts/files/770601/Response.txt)

[LineGraphDataTypeView.m.zip](https://github.com/danielgindi/Charts/files/770594/LineGraphDataTypeView.m.zip)

**Comment by liuxuan30** [comment]
Have you check out ChartsDemo - multiple bar chart?

**Comment by olivinesguerra** [comment]
yes sir. I play the demo code. It seems I cannot control the group location.

**Comment by liuxuan30** [comment]
how do you want to control?

**Comment by HPRaval** [comment]
i am facing same issue with 4 data set. Have you found any solution?
